Gemfile.lock

```diff
@@ -328,6 +328,7 @@ GEM
       xcpretty (~> 0.2, >= 0.0.7)
 
 PLATFORMS
+  arm64-darwin-21
```

Comment:
`arm64-darwin-21` corresponds to MacOS Monterey running on Apple Silicon - are there any downsides to having that added here?

Reply:
This is also fine! The iOS repo has a bunch more - https://github.com/RevenueCat/purchases-ios/blob/main/Gemfile.lock#L348-L353

Its basically just saying that `arm64-darwin-21` is okay to install on (which it is) 👍